tmt/steps/execute/__init__.py

```diff
@@ -395,6 +406,10 @@ def results(self) -> List["tmt.Result"]:
         """ Return test results """
         raise NotImplementedError
 
+    @property
+    def login_after_test(self) -> bool:
+        return self._login_after_test is not None
```

Comment:
This property does not seem to be used anywhere.

Reply:
Indeed. It can be deleted. It is just a public method for checking whether the login after the test is enabled. It is a residue from the previous implementation attempt.

Reply:
Ok, good, removed.